tools/deploy.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""Deploy API and UI to AppEngine.
+
+Usage:
+    deploy.py --project all-of-us-workbench-test --account $USER@pmi-ops.org
+"""
+
+import logging
+import subprocess
+
+from main_util import configure_logging, get_parser, get_confirmation
+import paths
+import gcloud_auth
+
+
+def deploy(args):
+  gcloud_auth.do_auth(args)
+
+  get_confirmation(
+      'Deploy to %r%s%s?' %
+      (args.project, ' (skip API)' if args.skip_api else '', ' (skip UI)' if args.skip_ui else ''))
+
+  if args.skip_api:
+    logging.info('Skipping deploy of API.')
+  else:
+    logging.info('Deploying API')
+    subprocess.check_call(
+        ['./gradlew', ':appengineDeploy'], cwd=paths.get_api_dir())
+
+  if args.skip_ui:
+    logging.info('Skipping deploy of UI.')
+  else:
+    logging.info('Building UI using %r', paths.get_ng())
+    subprocess.check_call([paths.get_ng(), 'build'], cwd=paths.get_ui_dir())
+    logging.info('Deploying UI')
+    subprocess.check_call(['gcloud', 'app', 'deploy'], cwd=paths.get_ui_dir())
```

Comment:
Rather than / in addition to using "gcloud set config project" and "gcloud auth login" above, we could use --project and --account flags in these calls. That would make it more explicit / ensure that some competing process on the box doesn't switch the project out from under us.